src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,54 @@
+package study;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+
+  @Test
+  @DisplayName("요구사항1: split 함수는 delimiter를 기준으로 문자열을 분리한다")
+  void split_based_on_delimiter() {
+    String given = "1,2";
+    String[] when = given.split(",");
+
+    assertThat(when).containsExactly("1", "2");
```

Comment:
```suggestion
    String input = "1,2";
    String[] actual = input.split(",");
    String[] expected = {"1", "2"};

    assertThat(actual).containsExactly(expected);
```

코드의 가독성을 높이고, 테스트의 의도를 명확하기 위해 이렇게 많이들 표현하는것 같아요 ^^